xcode/Subconscious/Shared/Components/Detail/DetailStackView.swift

```diff
@@ -23,39 +23,42 @@ struct DetailStackView<Root: View>: View {
                 tag: DetailStackAction.setDetails
             )
         ) {
-            root().navigationDestination(
-                for: MemoDetailDescription.self
-            ) { detail in
-                switch detail {
-                case .editor(let description):
-                    MemoEditorDetailView(
-                        app: app,
-                        description: description,
-                        notify: Address.forward(
-                            send: store.send,
-                            tag: DetailStackAction.tag
+            root()
+                .modifier(AppThemeToolbarViewModifier())
+                .modifier(AppThemeBackgroundViewModifier())
```

Comment:
Through a lot of trial and error I found (I think) the minimal set of locations to apply these modifiers without visual jank.